### ISSUE TITLE
Random number generation

### ISSUE THREAD

**Issue by pauliax** [vrexlab/dyverse-bounty#8]
contract KydyUtil
function getRand

`    uint256 private randomSeed_ = 0;`

Probably you are already aware of this issue, but I just want to mention that EVM is deterministic, so this random number generation is not so random. Also, do not think that private variables are really private, they could be read from the state.

**Comment by ouoabcde** [comment]
Yeah, I know about that. So we used `randomSeed_ += uint256(seedHash);` to change randomSeed_ all the time when calling `getRand()` function. And if you don't mind, I want to know your email address to contact you for rewards. Thank you again @pauliax :)

**Comment by pauliax** [comment]
> Yeah, I know about that. So we used `randomSeed_ += uint256(seedHash);` to change randomSeed_ all the time when calling `getRand()` function. And if you don't mind, I want to know your email address to contact you for rewards. Thank you again @pauliax :)

My email is: pauliax6 (at) gmail (dot) com

**Comment by ouoabcde** [comment]
Hello @pauliax ! We've opened our dapp **Dyverse** beta version on the Mainnet. Come and have a look! Here's the [link](http://dyver.se/).

**Comment by pauliax** [comment]
@ouoabcde looks pretty neat!

**Comment by pauliax** [comment]
Hello, when the rewards will be distributed?
